template/vars/config.sample.yaml

```
@@ -0,0 +1,59 @@
+---
+#
+# Bootstrap configuration - This file is gitignored
+#
+
+# Github username (e.g. onedr0p)
+bootstrap_github_username:
+# Github repository (e.g. flux-cluster-template)
+bootstrap_github_repository_name:
+# Age Public Key (e.g. age15uzrw396e67z9wdzsxzdk7ka0g2gr3l460e0slaea563zll3hdfqwqxdta)
+bootstrap_age_public_key:
+# Choose your timezone (e.g. America/New_York)
+bootstrap_timezone: Etc/UTC
+# Email you want to be associated with the ACME account
+bootstrap_acme_email:
+
+# Flux github webhook token (openssl rand -hex 12)
+bootstrap_flux_github_webhook_token:
+# Grafana admin password (openssl rand -hex 12)
+bootstrap_grafana_admin_password:
+# Weave GitOps dashboard password (update 'changeme')
+bootstrap_weave_gitops_admin_password: "{{ 'changeme' | password_hash('bcrypt') }}"
```

does it make sense to move the filter to the template?
```suggestion
bootstrap_weave_gitops_admin_password: "changeme"
```

---

Would I need to use another var to store the encrypted value? I was trying to keep it simple there.

---

I get your point, I was just thinking if it's better to have the same UX as with other passwords and tokens above.

---

I'll give it a shot because I get what you mean now :D

---

Ok, added that. thanks for the suggestion!